src/Dhadgar.Nodes/README.md

```
@@ -537,192 +217,184 @@ Nodes will have a state machine for tracking their lifecycle.
+| GET | `/` | Get reservation by token |
+| POST | `/claim` | Claim reservation with server ID |
+| DELETE | `/` | Release reservation |
 
-### Database Indexes (Planned)
+---
+
+## Database Schema
+
+### Primary Entities
+
+**Node**
```

markdownlint-cli2 (0.20.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)